README.md

```diff
@@ -205,6 +205,18 @@ To use the rake tasks, `require puppet-strings/rake_tasks` in your `Rakefile`:
 require 'puppet-strings/rake_tasks'
 ```
 
+By default, the rake task `strings:generate` will scan the manifests and lib directory from your single module. If you need to document a complete, or part of a, puppet tree, you can use `puppet-strings/rake_tasks/generate`. This allows you to set the paths and optionally patterns to include.
+
```

Comment:
Sorry, still not clear to me. Would I use this instead of `strings::generate`, or do I use this to set the paths and optional patterns to include so that the existing generate uses those options.
Perhaps my confusion comes from ignorance of how you use `strrings::generate` in the first place - but if I am confused others are probably as well.

(Think "optionally patterns to include" reads better as "optionally the patterns to include").